docs/user-experience/navigation.md

```diff
@@ -0,0 +1,22 @@
+# Navigation
+
+**Menu Structure.** Place your product navigation elements within the existing WooCommerce taxonomy.
+
+Examples:
+
+- If your extension is extending a component within WooCommerce, it should live within either the Extensions navigation drawer (in Woo Express stores), or directly within that category’s section.
+
+Extensions drawer (Woo Express)
+![Navigation extensions drawer](https://woo-docs-multi-com.go-vip.net/docs/wp-content/uploads/sites/3/2024/01/Image-1224x572-1.png)
+
+![Navigation category](https://woo-docs-multi-com.go-vip.net/docs/wp-content/uploads/sites/3/2024/01/Image-1242x764-1.png)
```

Comment:
This image has the title "Category sub-navigation", consistent to the "Extensions drawer" title above.